src/test/java/org/prebid/server/bidder/smaato/SmaatoBidderTest.java

```diff
@@ -184,6 +185,33 @@ public void makeHttpRequestsShouldModifyRequestSite() {
                 .containsOnly("keywords");
     }
 
+    @Test
+    public void makeHttpRequestsShouldModifyRequestApp() {
```

Comment:
In assert part, u checks only publisher ID, not whole app

Reply:
That's right. App would contain only `publisherId`: https://github.com/prebid/prebid-server/pull/1767/files#diff-edc0defc783203ca8071cac6a663d13097eed50ab60b283571aa10bffcb95781R104

Reply:
So makeHttpRequestsShouldModifyRequestAppPublisherId